pkgs/by-name/st/streamcontroller/package.nix

```diff
@@ -0,0 +1,218 @@
+{
+  stdenv,
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  copyDesktopItems,
+  makeDesktopItem,
+  wrapGAppsHook4,
+  gobject-introspection,
+  libadwaita,
+  libportal,
+  libportal-gtk4,
+  xdg-desktop-portal,
+  xdg-desktop-portal-gtk,
+}:
+let
+  streamcontroller-plugin-tools = python3Packages.buildPythonPackage rec {
```

Comment:
We should really expose this package otherwise applying overlays to it is unnecessarily hard.